decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './Nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
```

Comment:
- [ ] Please make sure to set up the association between the `Namebale` class and the `Decorator` class using `attr_accessor`. 🙏